Misc recipe fixes

JuiceyBeans opened the pull request:
Tons of fixes for Kube stuff
I'm too tired to fill this out lmao

JuiceyBeans committed:
Misc fixes

JuiceyBeans committed:
Raw iesnium namespace fix

JuiceyBeans committed:
Archwood is kil 🗣️

JuiceyBeans committed:
Remove dilute nitric acid distillation recipe for singleblock

DT autogens recipe for singleblocks anyway

JuiceyBeans committed:
Fix netherite space suit canning recipe

JuiceyBeans committed:
Fix corporea index recipe

JuiceyBeans committed:
Remove empty removals

JuiceyBeans committed:
Fix Olive Oil naming

JuiceyBeans committed:
Make fryer have 6 slots so it don't look janky

JuiceyBeans committed:
Update RecipeTypes.js

JuiceyBeans committed:
Remove stonecutter recipes by Aether, Undergarden

Ghostipedia commented:
At some point make a shorthanded changelog of all of the tweaks so we can put it in the big changelog, but merging for now :)

JuiceyBeans commented:
Full changelog for reference:
- Fixed canning recipe for Ad Astra space suit
- Added canning recipe for netherite space suit
- Fixed space boots recipe
- Fixed ME chest recipe
- Fixed flawless certus quartz in world transformation recipe
- Fixed ring of chordata recipe
- Fixed corporea index recipe
- Removed duplicate recipes for dilute nitric acid distillation 
- Ethylene oxide can only be made in LCRs now
- Bacon requires circuit 2 in slicer now
- Fixed guardian slicing recipe
- Fixed nether brick dust maceration recipe
- Simplified archwood log -> plank recipes
- Removed duplicate smelting recipes for iron, copper, gold, medium
- Removed stonecutter crafting recipe from aether and undergarden stone types
- Merged GT iesnium dust and raw ore with Occultism's
- Fixed rapier recipe
- Renamed Olive oil -> Olive Oil
- Increased IO slots for some recipe types so they work
